settings:
  data_path: "/data/intel_d435i/calib_data"
  results_path: "/data/intel_d435i/calib_data/calib_results.yaml"
  imshow: true

calib_target:
  target_type: 'aprilgrid'  # Target type
  tag_rows: 6               # Number of rows
  tag_cols: 6               # Number of cols
  tag_size: 0.088           # Size of apriltag, edge to edge [m]
  tag_spacing: 0.3          # Ratio of space between tags to tagSize
                            # Example: tagSize=2m, spacing=0.5m --> tagSpacing=0.25[-]

cam0:
  resolution: [640, 480]
  lens_hfov: 90
  lens_vfov: 60
  camera_model: "pinhole"
  distortion_model: "radtan"
